### PR TITLE
<SkeletonGroup/> - created new skeleton components

### PR DESCRIPTION
**Pull request by mharsat**
### 🔦 Summary

New components:

- SkeletonGroup
- SkeletonLine
- SkeletonCircle
- SkeletonRectangle

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/SkeletonGroup/SkeletonGroup.uni.driver.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { BaseUniDriver } from 'wix-ui-test-utils/unidriver';
+export interface SkeletonGroupUniDriver extends BaseUniDriver {
+  hasChildernSkin(): Promise<boolean>;
```
Comment (EgoziE) [inline src/SkeletonGroup/SkeletonGroup.uni.driver.d.ts:3]: Skin is a pure visual prop, therefore there's no need to test it programatically.
Visual tests are enough

File: src/SkeletonRectangle/docs/index.story.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```
Comment (EgoziE) [inline src/SkeletonRectangle/docs/index.story.js:1]: A single story page is enough for all Skeleton components.
They are not complex enough to have a separate story page.

File: src/colors.scss [hunk 3]
```diff
@@ -16,7 +16,9 @@ $D10-30: rgba($D10, 0.3);
 $D10-42: rgba($D10, 0.42);
 $D10-66: rgba($D10, 0.66);
 
+$D40-20: rgba($D40, 0.2);
```
Comment (EgoziE) [inline src/colors.scss:19]: This file is about to be deprecated, no need to add colors here

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6141

File: src/SkeletonRectangle/SkeletonRectangle.st.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
+:import {
+  -st-from: '../Foundation/stylable/colors.st.css';
+  -st-named: G30, R30;
+}
```
Comment (EgoziE) [inline src/SkeletonRectangle/SkeletonRectangle.st.css:4]: Not used anywhere

File: src/SkeletonCircle/SkeletonCircle.st.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
+:import {
+  -st-from: '../Foundation/stylable/colors.st.css';
+  -st-named: G30, R30;
+}
```
Comment (EgoziE) [inline src/SkeletonCircle/SkeletonCircle.st.css:4]: Not used anywhere

File: src/SkeletonLine/SkeletonLine.st.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
+:import {
+  -st-from: '../Foundation/stylable/colors.st.css';
+  -st-named: G30, R30;
+}
```
Comment (EgoziE) [inline src/SkeletonLine/SkeletonLine.st.css:4]: Not used anywhere

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Comment by assafzi** [comment]
Approved by UX.
Looks great.
Thanks!

File: src/Sidebar/index.d.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import SidebarBackButton from '../SidebarBackButton';
+import SidebarBackButton from "../SidebarBackButton";
```
Comment (EgoziE) [inline src/Sidebar/index.d.ts:2]: looks like a mistake :)

File: src/SkeletonGroup/SkeletonGroup.st.css [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+@keyframes wave {
+  0% {
+    left: -200px;
+  }
+
+  /*60% {
+    left: -200px;
+  }*/
```
Comment (EgoziE) [inline src/SkeletonGroup/SkeletonGroup.st.css:30]: If it's not needed, can be deleted.

File: src/SkeletonGroup/docs/index.story.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+  componentProps: {
+    skin: 'light',
+    children: exampleLayout,
+  },
+
+  exampleProps: {
+    skin: ['light', 'dark'],
+    children: childrenProps,
+  },
+
```
Comment (EgoziE) [inline src/SkeletonGroup/docs/index.story.js:70]: SkeletonGroup needs to have a background, use this pattern to add a background:
```
componentWrapper: ({ component }) => (...)
```
Take a look at `Palette` component for reference.

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by EgoziE — APPROVED** [review]
@bamai Looks good, let's merge it when all tests pass!

**Pull request merged**